server.js

```diff
@@ -1,12 +1,19 @@
 require("dotenv").config();
```

Comment:
note that this ALWAYS reads `.env` and not `.env.development`, regardless of what NODE_ENV is set